cms/src/components/content/EditContentBasic.vue

```diff
@@ -307,10 +308,10 @@ const publishStatus = computed({
                     <LButton
                         type="button"
                         name="6"
-                        variant="custom"
+                        variant="secondary"
                         class="flex-1"
                         size="lg"
```

Comment:
Please remove this one, as it's the only one on size `lg` and it causes the other ones to increase in size as well. The base size is fine I think.

```suggestion
```